torch_xla/distributed/xrt_init.py

```diff
@@ -0,0 +1,241 @@
+# Copyright Amazon Web Services and its Affiliates. All Rights Reserved.
+# ==============================================================================
+
```

Comment:
I don't think `yapf` is catching this, but can you also adjust the indent to two spaces to match the rest of the repository?

Reply:
will do ...thanks!